js/components/ReportSummaryForm.vue

```diff
@@ -186,7 +203,8 @@ export default {
     },
 
     clearErrors() {
-      this.errors = [];
```

Comment:
Completely replacing the `errors` object causes issues where dependent computed properties don't update, so this mutates its properties instead.